File: MPI/mpicheck/tests/functions/2.0/MPI_Type_free_keyval.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int *var_0;
    int ret;
    /* calls */
    ret = MPI_Type_free_keyval(var_0);
    ret = PMPI_Type_free_keyval(var_0);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Type_free_keyval.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
        call mpi_type_free_keyval(var_0, var_1)
       call pmpi_type_free_keyval(var_0, var_1)
        end program main
    